spec/date-rule.js

```diff
@@ -32,21 +32,14 @@ describe('date rule', function() {
   });
 
   it('should fail for incorrect date format', function() {
-    var asserts = [
```

Comment:
@skaterdav85 Would you prefer the it block to have 4 different assertions, or would you prefer for it blocks, with one assertion each?